get-content: add page (#4687)
tldr-pages/tldr: pages/windows/get-content.md

# Get-Content

> Get the content of the item at the specified location.
> More information: <https://docs.microsoft.com/powershell/module/microsoft.powershell.management/get-content>.

> - Get the content of a file:

Reviewer: ```suggestion
- Display the content of a file:
```

> `Get-Content -Path {{file_name}}`

Reviewer: We use this template:
```suggestion
`Get-Content -Path {{path/to/file}}`
```

> - Limit the number of lines returned by Get-Content:

Reviewer: ```suggestion
- Display the first few lines of a file:
```

> `Get-Content -Path {{file_name}} -TotalCount {{number_of_lines}}`

Reviewer: ```suggestion
`Get-Content -Path {{path/to/file}} -TotalCount {{count}}`
```

> - Get the content of the file and keeps it open to print each new line:

Reviewer: We can't use `keeps` because we shouldn't use `present simple`. I think this is better:
```suggestion
- Display the content of the file and keep reading it until `Ctrl + C`:
```

Reviewer 2: What about this, @navarroaxel?

```suggestion
- Display the content of the file and keep reading from it until `Ctrl + C` is pressed:
```

> `Get-Content -Path {{file_name}} -Wait`

Reviewer: ```suggestion
`Get-Content -Path {{path/to/file}} -Wait`
```
